### ISSUE TITLE
SQS Queues now accept Tags

### ISSUE THREAD

**Issue by tomdionysus** [martysweet/cfn-lint#214]
SQS Queues (AWS::SQS::Queue) now - my assumption is that they didn't previously - accept tags, leading to an incorrect warning:

```
...
    "TestResources": {
      "Type": "AWS::SQS::Queue",
      "Properties": {
        "ContentBasedDeduplication": true,
        "DelaySeconds": 0,
        "QueueName": "workqueue",
        "Tags": [
          {
            "Key": "Name",
            "Value": "xxxxx-test-sqs-workqueue"
          }
        ],
        "VisibilityTimeout": 3600
      }
    },
...
```
Output:
```
0 infos
0 warn
1 crit
Resource: Resources > TestResources > Properties > Tags
Message: Tags is not a valid property of AWS::SQS::Queue
Documentation: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-sqs-queues.html
```

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-sqs-queues.html#cfn-sqs-queue-tags

I'm happy to submit a PR to fix on acceptance.

**Comment by martysweet** [comment]
Hi @tomdionysus, this should just be a cfn specification update - as it was reInvent last week I should probably push an update! Will do one now.

**Comment by martysweet** [comment]
@tomdionysus Can you confirm this is fixed in v1.9.3? Thanks!

**Comment by tomdionysus** [comment]
Sorry @martysweet it seems it's still reporting it. 

```
0 infos
0 warn
1 crit
Resource: Resources > TestResources > Properties > Tags
Message: Tags is not a valid property of AWS::SQS::Queue
Documentation: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-sqs-queues.html

Template invalid!
```

I've confirmed i'm definitely using 1.9.3

```
  "dependencies": {
    "cfn-lint": "^1.9.3",
...
```

**Comment by martysweet** [comment]
Hi @tomdionysus,

I've just been back through the versions and it appears this should of worked since v1.8.1, with versions before that reporting `Tags is not a valid property of AWS::SQS::Queue`.

Could you check the version running with `cfn-lint --version`?
